web/profiles/joinup/src/Form/UserMultipleCancelConfirm.php

```diff
@@ -64,30 +63,40 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form = parent::buildForm($form, $form_state);
 
     // Loop through all the accounts that are going to be deleted and check
-    // if they are sole owners of collections.
+    // if they are sole owners of groups.
     $build = [];
     foreach (Element::children($form['accounts']) as $user_id) {
       /** @var \Drupal\user\Entity\User $account */
       $account = $this->userStorage->load($user_id);
       if (empty($account)) {
         throw new \RuntimeException("User with id {$user_id} was not found.");
       }
-      $collections = $this->relationManager->getCollectionsWhereSoleOwner($account);
+      $groups = $this->relationManager->getGroupsWhereSoleOwner($account);
 
-      if ($collections) {
+      if ($groups) {
         $build[$account->id()] = [
           'warning' => [
-            '#markup' => $this->t('User @name cannot be deleted as it is currently the sole owner of these collections:', [
+            '#markup' => $this->t('User @name cannot be deleted as they are currently the sole owner of these groups:', [
               '@name' => $account->getAccountName(),
```

Comment:
This wrongly uses `->getAccountName()` which shows the username. But we want to show the real name, as we do everywhere on the site: `->getDisplayName()`. Should we change?